src/system/mixins/media-query.js

```diff
@@ -23,10 +23,15 @@ function initListener() {
   if (init) {
     return
   }
-  if (window && typeof window !== 'undefined') {
-    window.addEventListener('resize', updateWindowSize)
-    updateWindowSize()
+  try {
+    if (window && typeof window !== 'undefined') {
+      window.addEventListener('resize', updateWindowSize)
+      updateWindowSize()
+    }
+    init = true
+  } catch (err) {
     init = true
+    return false
```

Comment:
It looks like it breaks the media query function all together, when loaded with SSR. Wouldn't it make more sense to add checks for everything necessary for updateWindowSize to work?

Reply:
Im open for a better solution as that’s like a hack somehow. I was thinking if the media query should be used on the grid at all but just on the components. That would fix the jumping layout on ssr load I think. Any suggestions?